testfm/constants.py

```diff
@@ -60,30 +80,40 @@
 
 # Capsule repositories
 cap_68_repos = [
-    "rhel-7-server-satellite-tools-6.8-rpms",
     "rhel-7-server-satellite-capsule-6.8-rpms",
+    "rhel-7-server-satellite-maintenance-6-rpms",
 ] + common_repos
+
 cap_69_repos = [
-    "rhel-7-server-satellite-tools-6.9-rpms",
     "rhel-7-server-satellite-capsule-6.9-rpms",
+    "rhel-7-server-satellite-maintenance-6-rpms",
 ] + common_repos
+
 cap_610_repos = [
-    "rhel-7-server-satellite-tools-6.10-rpms",
     "rhel-7-server-satellite-capsule-6.10-rpms",
+    "rhel-7-server-satellite-maintenance-6-rpms",
 ] + common_repos
-cap_70_repos = [
-    "rhel-7-server-satellite-tools-7.0-rpms",
-    "rhel-7-server-satellite-capsule-7.0-rpms",
-] + common_repos
+
+cap_70_repos = (
+    [
+        "rhel-7-server-satellite-capsule-7.0-rpms",
+        "rhel-7-server-satellite-maintenance-7.0-rpms",
+    ]
+    + common_repos
+    if rhel7()
+    else [
+        "satellite-capsule-7.0-for-rhel-8-x86_64-rpms",
+        "satellite-maintenance-7.0-for-rhel-8-x86_64-rpms",
+    ]
+    + common_repos
+)
```

Comment:
Non blocking comment:
```suggestion
cap_70_repos = (
    [
        "rhel-7-server-satellite-capsule-7.0-rpms",
        "rhel-7-server-satellite-maintenance-7.0-rpms",
    ]
    if rhel7()
    else [
        "satellite-capsule-7.0-for-rhel-8-x86_64-rpms",
        "satellite-maintenance-7.0-for-rhel-8-x86_64-rpms",
    ]
) + common_repos
```
same for `sat_70_repos`

Reply:
Updated, request to re-review